docs/resources/assume_service_account_policy.md

```diff
@@ -10,7 +10,52 @@ order: 5
 
 Manages an Assume Service Account Policy in Scalr.
 
+## Example Usage
 
+### Github Actions
+
+```terraform
+data "scalr_workload_identity_provider" "github" {
+  url = "https://token.actions.githubusercontent.com"
```

Comment:
does a data-source support filtering by URL? or should we add by name for the sake of simplicity?